utensor_cgen/experimental/ugraph_util_functions.py

```diff
@@ -116,4 +121,27 @@ def replace_tensor(name, new_tensorInfo, ugraph):
     #outputs
     for i, t_info in enumerate(op_info.output_tensors):
       if(t_info.name == name):
-        op_info.output_tensors[i] = new_tensorInfo
\ No newline at end of file
+        op_info.output_tensors[i] = new_tensorInfo
+
+def update_tensor_op_names(graph):
```

Comment:
@dboyliao this logic enables us to backtrace without maintaining a state in the tensor-info